portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -18,6 +18,16 @@ definition {
 			value1 = "${keyword}");
 	}
 
+	macro changeToAC {
```

Comment:
hey @ViniLopes20 , the changeToAC macro is duplicated.

And please, move this macro to ACLogin.macro, renaming it to launchAC.